tiup/tiup-cluster-topology-reference.md

```diff
@@ -118,6 +120,43 @@ server_configs:
 
 上述配置指定了 TiDB 和 TiKV 的全局配置。
 
+### `component_versions`
+
+> **注意：**
+>
+> 对于 TiDB、TiKV、PD、TiCDC 等本身共用版本号的组件，没有完整的测试来保证它们能够在跨版本混部的场景下能够正常工作。请仅用于测试场景，或者在技术支持的帮助下进行操作。
+
+`component_versions` 用于指定某个组件的版本号。
+
+- 如果没有配置 `component_versions`，各个组件要么使用和 TiDB 集群一致的版本号（如 PD、TiKV），要么使用最新版本号（如 Alertmanager）。
+- 如果配置了该字段，对应的组件将会固定使用对应的版本，并且在后续的扩容和升级集群操作中都使用该版本。
```

Comment:
```suggestion
- 如果配置了该字段，相应的组件将固定使用指定的版本，并且在扩容或升级集群操作中继续使用该版本。
```

Reply:
I think it is more clear to remain old description